Add docs for Connect My Computer (#32798)
gravitational/teleport: docs/pages/connect-your-client/includes/connect-my-computer-prerequisites.mdx

- A macOS or Linux device.

> - Teleport Connect v14.1+, on the same major version or one version behind the proxy version. See

Reviewer: ```suggestion
- Teleport Connect v14.1+, on the same major version or one version behind the Teleport Proxy Service version. See
```

  [version compatibility](../../faq.mdx#version-compatibility).

> - A local Teleport user: you authenticate using credentials or passwordless login, you don't
>   use SSO to log in to the cluster.

Reviewer: ```suggestion
- A local Teleport user: you must authenticate using credentials or passwordless login, and not with SSO.
```
